src/components/wallets/WalletAccordion.tsx

```diff
@@ -0,0 +1,327 @@
+import { useState } from 'react'
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material'
+import styled from 'styled-components'
+import { DateTime } from 'luxon'
+
+import { theme } from '~/styles'
+import {
+  Avatar,
+  Button,
+  Icon,
+  Skeleton,
+  Status,
+  StatusEnum,
+  Tooltip,
+  Typography,
+} from '~/components/designSystem'
+import { useInternationalization } from '~/hooks/core/useInternationalization'
+import { Wallet, WalletStatusEnum } from '~/generated/graphql'
+
+interface WalletAccordionProps {
+  wallet: Omit<Wallet, 'updatedAt'>
+}
+
+const mapStatus = (type?: WalletStatusEnum | undefined) => {
+  switch (type) {
+    case WalletStatusEnum.Active:
+      return {
+        type: StatusEnum.running,
+        label: 'text_62da6ec24a8e24e44f812874',
+      }
+    default:
+      return {
+        type: StatusEnum.failed,
+        label: 'text_62da6ec24a8e24e44f8128b0',
+      }
+  }
+}
+
+export const WalletAccordion = ({ wallet }: WalletAccordionProps) => {
+  const status = mapStatus(wallet?.status)
+  let [creditAmountUnit = '0', creditAmountCents = '00'] = wallet.creditsBalance.split('.')
+  let [consumerCreditUnit = '0', consumerCreditCents = '00'] = wallet.consumedCredits.split('.')
+  const [isOpen, setIsOpen] = useState(wallet?.status === 'active')
+  const { translate } = useInternationalization()
+
+  return (
+    <Container id={`wallet-${wallet?.id}`}>
+      <StyledAccordion expanded={isOpen} onChange={(_, expanded) => setIsOpen(expanded)} square>
+        <Summary $isOpen={isOpen}>
+          <SummaryLeft>
+            <Button
+              variant="quaternary"
+              size="small"
+              icon={isOpen ? 'chevron-down' : 'chevron-right'}
+            />
+            <Avatar variant="connector">
+              <Icon name="wallet" color="dark" />
+            </Avatar>
+            <SummaryInfos $isLoading={false}>
```

Comment:
If it's `false` just don't add `$isLoading` here
And when it's `true` you can just do : <SummaryInfos $isLoading>

Reply:
Updated